modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/display/context/util/CETLabelUtil.java

```diff
@@ -50,6 +50,11 @@ else if (Objects.equals(
 
 			return "css";
 		}
+		else if (Objects.equals(
+					type, ClientExtensionEntryConstants.TYPE_THEME_SPRITEMAP)) {
+
+			return "svg";
```

Comment:
Small detail: as we're overriding the theme SVG spritemap, this label may say something in line with the other `theme-*` types of extension (lines 71 and 76), kinda `theme-svg` which would be translated as one of "Theme SVG Spritemap", "Theme SVG" or "Theme spritemap" in the language.properties